app/views/comments/review.js.erb

```diff
@@ -1,4 +1,10 @@
+<% comment_container_id = "#feedback_comment_#{@comment.id}" %>
+<% unread_comment_label_id = "#unread_comment_label_#{@comment.id}" %>
+<% read_comment_form_id = "#read_comment_form_#{@comment.id}" %>
 <% review_comment_list_item_id = "#review_comment_link_#{@comment.id}" %>
 <% reply_button_html = "#{j render('inbox/reply_button', feedback_comment: @comment) }" %>
 
+$j("<%= comment_container_id %>").removeClass("unreviewed").find("h4").find(".unreviewed").remove();
+$j("<%= unread_comment_label_id %>").remove();
+$j("<%= read_comment_form_id %>").remove();
```

Comment:
I think you can do both elements at once, like: `$j("<%= unread_comment_label_id %>, <%= read_comment_form_id %>").remove();`